### ISSUE TITLE
Fix S6967 FP: Raises when model has no validation attributes

### ISSUE THREAD

**Issue by martincook3** [SonarSource/sonar-dotnet#9349]
### Description

Rule [S6967](https://sonarsource.github.io/rspec/#/rspec/S6967/csharp) is firing for a model with no validation attributes.

### Repro steps

Simplified example:

```
[Controller]
public class MyController : ControllerBase
{
    [HttpPost]
    public async Task<IActionResult> CheckYourAnswers(CheckYourAnswersViewModel model)
    {
       // view is readonly
        return RedirectToAction("Confirmation");
    }
}

public class CheckYourAnswersViewModel
{
    public string Property1 { get; set; } = string.Empty;
    public AnotherViewModel Property2 { get; set; }
}

public class AnotherViewModel
{
    public string Property1 { get; set; }
}

```

### Expected behaviour

Rule [S6967](https://sonarsource.github.io/rspec/#/rspec/S6967/csharp) says:
| The rule will not raise issues if the model, or the model members, are not decorated with validation attributes

As neither model class has validation attributes the rule is not triggered.

### Actual behaviour

S6967 is reported.

### Known workarounds

Please provide a description of any known workarounds.

### Related information

* C#/VB.NET Plugins version
* Visual Studio version
* MSBuild / dotnet version
* SonarScanner for .NET version (if used)
* Operating System

**Comment by pavel-mikula-sonarsource** [comment]
Internal ticket [NET-1583](https://sonarsource.atlassian.net/browse/NET-1583)

[NET-1583]: https://sonarsource.atlassian.net/browse/NET-1583?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
